src/handlers/wildcard/weekly/action.ts

```diff
@@ -8,8 +8,8 @@ import { BotConfig } from "../../../types";
 import { getFallback } from "../../../utils/fallback";
 import { fetchImage } from "../../../utils/webAssets";
 import { weeklyConfig } from "../../../configs/weekly";
+import { wait } from "../../../helpers";
```

Comment:
This wait looks like a timeout which raises a flag. What is this for?

Reply:
that fn has been created by @Steveantor  originally. I am not 100% sure but I guess its being used to wait for 1 min between github api calls because of its rate limit.

Reply:
It would be better to read the rate limit header that's passed with the GitHub request and change behavior based on that number I think.